core/templates/domain/question/QuestionSummaryObjectFactory.ts

```diff
@@ -26,17 +26,17 @@ export interface IQuestionSummaryBackendDict {
 }
 
 export class QuestionSummary {
-  _questionId: string;
+  _id: string;
```

Comment:
Leave this as it was (_questionId) and perhaps update the backend dict accordingly. Better to be clearer about what entity has the id (the question, not the question summary).

Reply:
>Better to be clearer about what entity has the id (the question, not the question summary).

Got it! done!